website/docs/r/dbaas_logs_cluster.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_dbaas_logs_cluster"
+sidebar_current: "docs-ovh-resource-dbaas-logs-cluster"
+description: |-
+  Manage a existing dbaas logs cluster.
+---
+
+# ovh_dbaas_logs_cluster
+
+Reference a DBaaS logs cluster to manipulate ACL on a `DEDICATED` cluster
```

Comment:
I see only `standard` or `enterprise`, not "dedicated" ;-)
![image](https://user-images.githubusercontent.com/2543381/219033491-e6c74e16-8771-47a1-8dcf-8fd54b15b49d.png)

Reply:
`DEDICATED` is the value returned by the API

Reply:
ok :)